### ISSUE TITLE
[Enhancement]: Extra Narrator Filter

### ISSUE THREAD

**Issue by dcoffin88** [advplyr/audiobookshelf#2224]
### Describe the feature/enhancement

Would it be possible to add a Count filter on the narrator in library view to help filter books with a full cast vs. ones with a single narrator? Even just a toggle between narrator =1 and >1.

**Comment by Gru0810** [comment]
I agree but it would be better to implement narrator in the same way like author incl. picture, description and year of birthday

**Comment by bugreport1** [comment]
Being able to filter by:
- Solo narration
- Duo narration
- Cast narration (3+)
Would be greatly appreciated.
